R/check_description.R

```diff
@@ -221,7 +221,7 @@ check_license <- function(x = ".") {
 Please send a pull request if you need support for this license.",
       this_desc$get_field("License")
     )[
-      !current_license %in% c("GPL-3", "MIT")
+      !current_license %in% c("GPL-3", "MIT + file LICENSE")
```

Comment:
Does this works? That would make 31ac63f156d obsolete.

```suggestion
      !current_license %in% c("GPL-3", "MIT + file LICENSE.md")
```

Reply:
It needs to stay as is. In case of a MIT license, there will be a LICENSE.md file and a LICENSE file. The LICENSE.md file has the license text (with copyright holder and year filled in). The LICENSE file has two lines (e.g.):

```
YEAR: 2022
COPYRIGHT HOLDER: Research Institution for Nature and Forest
```

This construction is necessary to pass CRAN tests (see also the links and explanation in https://github.com/inbo/checklist/pull/93#issue-1375720806)